<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navmap - Connect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="12" column="1">
      <widget class="QLabel" name="labelConnectUpdateRateXp">
       <property name="text">
        <string>&amp;Update time interval:</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxConnectUpdateRateFsx</cstring>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLabel" name="labelConnectPort">
       <property name="text">
        <string>&amp;Port:</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxConnectPort</cstring>
       </property>
      </widget>
     </item>
     <item row="20" column="0" colspan="4">
      <widget class="Line" name="lineBottom">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="labelConnectUpdateRateFsx">
       <property name="text">
        <string>&amp;Update time interval:</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxConnectUpdateRateFsx</cstring>
       </property>
      </widget>
     </item>
     <item row="19" column="2">
      <widget class="QSpinBox" name="spinBoxConnectPort">
       <property name="toolTip">
        <string>Add the &lt;span style=&quot;color: #ff0000; font-weight:bold&quot;&gt;port&lt;/span&gt; that Little Navconnect shows on startup.
Use your X-Plane computer name if you are running the Little Xpconnect X-Plane plugin.</string>
       </property>
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>51968</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="4">
      <widget class="Line" name="lineFsxXp">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QSpinBox" name="spinBoxConnectUpdateRateFsx">
       <property name="toolTip">
        <string>Allowed range is 50 milliseconds up to 1 second.
Little Navmap fetches data from the flight simulator using this time interval.
Lower values will result in a more fluid aircraft movement but can potentially induce stutters.</string>
       </property>
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>50</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="18" column="3">
      <widget class="QPushButton" name="pushButtonConnectDeleteHostname">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Delete the current hostname or IP address from the list.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../littlenavmap.qrc">
         <normaloff>:/littlenavmap/resources/icons/cancel.svg</normaloff>:/littlenavmap/resources/icons/cancel.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLabel" name="labelConnectHostname">
       <property name="text">
        <string>&amp;Hostname or IP address:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxConnectHostname</cstring>
       </property>
      </widget>
     </item>
     <item row="17" column="0" colspan="4">
      <widget class="QRadioButton" name="radioButtonConnectRemote">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Connect to FSX, Prepar3D or X-Plane on a remote computer running the Little Navconnect program.</string>
       </property>
       <property name="text">
        <string>Connect to a &amp;remote flight simulator via Little Navconnect</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupConnectType</string>
       </attribute>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="QRadioButton" name="radioButtonConnectDirectFsx">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Connect directly to a FSX or a Prepar3D flight simulator on the same computer using SimConnect.</string>
       </property>
       <property name="text">
        <string>Connect directly to a &amp;local FSX or Prepar3D flight simulator</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupConnectType</string>
       </attribute>
      </widget>
     </item>
     <item row="18" column="2">
      <widget class="QComboBox" name="comboBoxConnectHostname">
       <property name="toolTip">
        <string>Add the &lt;span style=&quot;color: #0000ff; font-weight:bold&quot;&gt;hostname&lt;/span&gt; or &lt;span style=&quot;color: #0000ff; font-weight:bold&quot;&gt;IP address&lt;/span&gt; that Little Navconnect shows on startup.
Use your X-Plane computer name if you are running the Little Xpconnect X-Plane plugin.</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="maxCount">
        <number>10</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="4">
      <widget class="QRadioButton" name="radioButtonConnectDirectXp">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Connect directly to X-Plane on the same computer using the Little Xpconnect plugin.</string>
       </property>
       <property name="text">
        <string>Connect directly to a local &amp;X-Plane simulator</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupConnectType</string>
       </attribute>
      </widget>
     </item>
     <item row="9" column="1" colspan="3">
      <widget class="QCheckBox" name="checkBoxConnectFetchAiShipFsx">
       <property name="toolTip">
        <string>Deselect to disable fetching and transmission of AI or multiplayer ships</string>
       </property>
       <property name="text">
        <string>Fetch AI or multiplayer &amp;ships</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="16" column="0" colspan="4">
      <widget class="Line" name="lineXpRemote">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QSpinBox" name="spinBoxConnectUpdateRateXp">
       <property name="toolTip">
        <string>Allowed range is 50 milliseconds up to 1 second.
Little Navmap fetches data from the flight simulator using this time interval.
Lower values will result in a more fluid aircraft movement but can potentially induce stutters.</string>
       </property>
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>50</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="3">
      <widget class="QCheckBox" name="checkBoxConnectFetchAiAircraftFsx">
       <property name="toolTip">
        <string>Deselect to disable fetching and transmission of AI or multiplayer aircraft</string>
       </property>
       <property name="text">
        <string>Fetch AI or multiplayer &amp;aircraft</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="1" colspan="3">
      <widget class="QCheckBox" name="checkBoxConnectFetchAiAircraftXp">
       <property name="toolTip">
        <string>Deselect to disable fetching and transmission of AI or multiplayer aircraft</string>
       </property>
       <property name="text">
        <string>Fetch AI or multiplayer &amp;aircraft</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="14" column="1" colspan="3">
      <widget class="QCheckBox" name="checkBoxConnectFetchAiShipXp">
       <property name="toolTip">
        <string>Deselect to disable fetching and transmission of AI or multiplayer ships</string>
       </property>
       <property name="text">
        <string>Fetch AI or multiplayer &amp;ships</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxConnectOnStartup">
     <property name="toolTip">
      <string>Automatically connect to given address and port of a remote simulator or to the local flight simulator.
You have to click &quot;Connect&quot; to start the constant connection checking.
Unchecking this will stop all automatic connection attempts immediately.</string>
     </property>
     <property name="text">
      <string>Connect &amp;automatically</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QDialogButtonBox" name="buttonBoxConnect">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonConnectDirectFsx</tabstop>
  <tabstop>checkBoxConnectFetchAiAircraftFsx</tabstop>
  <tabstop>checkBoxConnectFetchAiShipFsx</tabstop>
  <tabstop>radioButtonConnectRemote</tabstop>
  <tabstop>checkBoxConnectOnStartup</tabstop>
 </tabstops>
 <resources>
  <include location="../../littlenavmap.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupConnectType"/>
 </buttongroups>
</ui>
